tests/unit/tf/blocks/core/test_tabular.py

```diff
@@ -1,28 +1,42 @@
-import merlin.models.tf as ml
+import merlin.models.tf as mm
+from merlin.schema import Schema, Tags
 
 
 def test_filter_features(tf_con_features):
     features = ["a", "b"]
-    con = ml.Filter(features)(tf_con_features)
+    con = mm.Filter(features)(tf_con_features)
 
     assert list(con.keys()) == features
 
 
+def test_filter_call(tf_con_features):
+    assert mm.Filter([])({}) == {}
+    assert mm.Filter([])(tf_con_features) == {}
+    assert mm.Filter(["unknown"])(tf_con_features) == {}
+
+    assert mm.Filter(Schema())({}) == {}
+    assert mm.Filter(Schema())(tf_con_features) == {}
+    assert mm.Filter(Schema(["unknown"]))(tf_con_features) == {}
+
+    assert mm.Filter(Tags.CONTINUOUS)({}) == {}
+    assert mm.Filter(Tags.CONTINUOUS).set_schema(Schema(["unknown"]))(tf_con_features) == {}
```

Comment:
This tag version is a bit different to the others because passing a Tag to the constructor doesn't provide enough information to figure out the column names.